rotkehlchen/tests/api/test_settings.py

```diff
@@ -35,7 +35,8 @@
 )
 
 
-def test_cached_settings(rotkehlchen_api_server, username):
+@pytest.mark.mock_rotki_settings(False)
```

Comment:
This is something that is really only used here. Instead you can use the fixture that we have to set settings. An example of how to use it is

```
@pytest.mark.parametrize('db_settings', [
    {'pnl_csv_with_formulas': False},
])
```

and you can modify some of them and test them

Reply:
I took this approach, but ended up having to create a new fixture to control the relevant flag